router/internal/requestlogger/access_log_expression.go

```diff
@@ -0,0 +1,53 @@
+package requestlogger
+
+import (
+	"fmt"
+	"github.com/expr-lang/expr/vm"
+	exprlocal "github.com/wundergraph/cosmo/router/internal/expr"
+	"github.com/wundergraph/cosmo/router/pkg/config"
+)
+
+type ExpressionAttribute struct {
+	Key     string
+	Default string
+	Expr    *vm.Program
+}
+
+func GetAccessLogConfigExpressions(attributes []config.CustomAttribute) ([]ExpressionAttribute, error) {
+	exprSlice := make([]ExpressionAttribute, 0)
+	for _, sAttribute := range attributes {
+		exprString := sAttribute.ValueFrom.Expression
+		if sAttribute.ValueFrom == nil || exprString == "" {
+			continue
+		}
+
+		err := exprlocal.ValidateAnyExpression(exprString)
```

Comment:
Validate against nil and a few other types, we can add / remove types later as necessary